content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -723,11 +714,8 @@ Not all Persistent Volume types support mount options.
 
 The following volume types support mount options:
 
-* `awsElasticBlockStore`
-* `azureDisk`
 * `azureFile`
 * `cephfs`
-* `cinder` (**deprecated** in v1.18)
 * `gcePersistentDisk`
```

Comment:
gcepd is removed.

Reply:
According to https://github.com/kubernetes/website/pull/42302#discussion_r1282723644, GCE PD is still in Kubernetes 1.27